cmd/talosctl/acompat/acompat.go

```diff
@@ -0,0 +1,14 @@
+// This Source Code Form is subject to the terms of the Mozilla Public
+// License, v. 2.0. If a copy of the MPL was not distributed with this
+// file, You can obtain one at http://mozilla.org/MPL/2.0/.
+
+// Package acompat provides compatibility with gRPC 1.67.0 and later.
+package acompat
+
+import "os"
+
+func init() {
+	if err := os.Setenv("GRPC_ENFORCE_ALPN_ENABLED", "false"); err != nil {
```

Comment:
this will unfortunately be something we need more of: as the Talos cluster is upgraded, worker nodes talk to `trustd` on the controlplane, and also `apid`s talk to each other. So if 1.9 machine tries to talk to 1.8 machine, and ALPN issue will pop up

Reply:
Done in https://github.com/siderolabs/talos/pull/9504